Projects/Console Monsters/Statics.cs

```diff
@@ -9,6 +9,7 @@ public static class Statics
 	public static bool DisableBattleTransition { get; set; } = false;
 	public static bool FirstTimeLaunching { get; set; } = true;
 	public static bool AudioEnabled { get; set; } = true;
+	public static bool FastText { get; set; } = true;
```

Comment:
This property is not influencing anything. `SlowText` is not used anywhere yet but it provides some functionality. I would introduceit in PR which makes an use of `SlowText` somewhere.

Reply:
It is being used now, and was just for the future since I had an idea where I was going with it.
https://github.com/ZacharyPatten/dotnet-console-games/blob/945918641e84ce49f79f3f780886d3158a85b931/Projects/Console%20Monsters/BattleSystem.cs#L147-L157